builtin/rebase.c

```diff
@@ -61,11 +73,11 @@ static int finish_rebase(struct rebase_options *opts)
 
 	delete_ref(NULL, "REBASE_HEAD", NULL, REF_NO_DEREF);
 	apply_autostash();
+	close_all_packs(the_repository->objects);
 	/*
 	 * We ignore errors in 'gc --auto', since the
 	 * user should see them.
 	 */
-	close_all_packs(the_repository->objects);
```

Comment:
I think we should move this line-move to the earlier commit that introduced that code.